catalog/task/update-ocp-tag/0.1/update-ocp-tag.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: update-ocp-tag
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update pull-spec tag with
+    valid OCP version from get-ocp-version task.
+  params:
+    - name: fromIndex
+      description: The source Index image (catalog of catalogs) FBC fragment
+      type: string
+    - name: targetIndex
+      description: Index image (catalog of catalogs) the FBC fragment will be added to
+      type: string
+    - name: binaryImage
+      description: OCP binary image to be baked into the index image
+      type: string
+    - name: ocpVersion
+      description: OCP version tag to replace the current set tags on index images
+      type: string
+  results:
+    - name: updated-fromIndex
+      description: source Index image (catalog of catalogs) FBC fragment with updated tag
+    - name: updated-targetIndex
+      description: Index image (catalog of catalogs) the FBC fragment will be added to with updated tag
+    - name: updated-binaryImage
+      description: OCP binary image to be baked into the index image with updated tag
+  steps:
+    - name: update-ocp-tag
+      image: quay.io/hacbs-release/release-utils:4d8649dbb2b626f5fe9f4ff83c1bc3be268fad31
+      script: |
+       #!/usr/bin/env sh
+
+        # Function to replace tag in an image
+        replace_tag() {
+            local updatedImage="${1%:*}:$(params.ocpVersion)"
+            echo "$updatedImage"
+        }
+
+        # Access the updated image
+        updatedFromIndex=$(replace_tag "$(params.fromIndex)")
+        updatedTargetIndex=$(replace_tag "$(params.targetIndex)")
+        updatedBinaryImage=$(replace_tag "$(params.binaryImage)")
+
+        echo "Updated values"
+        echo -n "$updatedFromIndex" | tee $(results.updated-fromIndex.path)
+        echo    # Print a new line
+        echo -n "$updatedTargetIndex" | tee $(results.updated-targetIndex.path)
+        echo    # Print a new line
+        echo -n "$updatedBinaryImage" | tee $(results.updated-binaryImage.path)
```

Comment:
I'd suggest printing another line after this as well - it's always good to end any output with "\n".